src/Miso/Html/Element.hs

```diff
@@ -51,7 +51,9 @@ module Miso.Html.Element
     , canvas_
     , math_
     , script_
```

Comment:
`script_` is dangerous, even something as simple as:
```
script_ [] [text "console.log('<test>')"]
```

Fails with:
```
SyntaxError: expected expression, got '&' localhost:3003:1:12
VTree differed from node Object { type: "vtext", text: "console.log('<test>')" } #text "console.log(&#39;&lt;test&gt;&#39;)" all.js:695:152
Could not copy DOM into virtual DOM, falling back to diff all.js:694:395
<test> line 685 > scriptElement:1:9
```

I'm tempted to reclaim the name `script_` for the new version that I'm adding. What do you think?

Reply:
I'd say go for it.

Reply:
Then I'll have `style_` for consistency as well.